src/open_inwoner/pdc/models/faq.py

```diff
@@ -24,7 +26,18 @@ class Question(OrderedModel):
         null=True,
     )
     question = models.CharField(verbose_name=_("Vraag"), max_length=250)
-    answer = models.TextField(verbose_name=_("Antwoord"))
+    answer = ProsemirrorModelField(
+        _("Antwoord"),
+        allowed_node_types=[NodeType.PARAGRAPH],
```

Comment:
It seems this field sometimes uses blockquote:

<img width="1673" height="330" alt="Image" src="https://github.com/user-attachments/assets/922d609e-0ad8-467d-90a6-f6a0756392e5" />

```suggestion
        _("Antwoord"),
        allowed_node_types=[NodeType.PARAGRAPH, NodeType.BLOCKQUOTE],
```